index.js

```diff
@@ -55,26 +53,16 @@ async function accessSpreadsheet(status, url) {
       'max-col': sheet.colCount,
       'return-empty': true,
   })
+  
+  const index = cells.findIndex(cell => cell.value === url);
 
-  // Find the feature we want to update
-  for (const cell of cells) {
-      if (cell.value === url) {
-        data = cells.indexOf(cell)
-      }
-  }
-
-  updateStatusCell(cells, data, status)
+  updateStatusCell(cells, index, status)
 }
 
 // Update the next empty cell with the status from the GitHub comment
-async function updateStatusCell(cells, data, status){
-  do {
-    data++
-  }
-
-  // Get the last empty cell in the range
-  while (cells[data].value)
-  var cell = cells[data];
+async function updateStatusCell(cells, index, status) {
+  const emptyIndex = cells.findIndex(cell => !cell.value);
```

Comment:
Again, no need for a `do/while` loop or anything, we can use the `findIndex` method again instead :100: